lib/CMakeLists.txt

```diff
@@ -112,6 +111,13 @@ if (UNIX)
         set(LIBFACTER_STANDARD_SOURCES ${LIBFACTER_STANDARD_SOURCES} "src/util/posix/scoped_bio.cc")
     endif()
 
+    if (NOT AIX)
```

Comment:
Yeah. AIX has its own implementation of the kernel resolver that uses `oslevel` instead of uname output :/